composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "php": "^5.3 || ^7.0",
+        "php": "^7.1",
```

Comment:
PHP 7.1 will allow new cool features. We shouldn't restrain ourself while developing the first stable release.